praw/models/helpers.py

```diff
@@ -105,7 +105,7 @@ def now(self) -> Optional[LiveThread]:
 
         .. code-block:: python
 
-        thread = reddit.live.now()  # LiveThread object or None
+            thread = reddit.live.now()  # LiveThread object or None
```

Comment:
It's not a big deal, but I usually align these blocks under the `c` so with three additional spaces.

Reply:
I always use a tab/4 spaces since my IDE works well with 4 spaces.

Reply:
That's fine. There will just be some inconsistency here.